tools/build/__fixtures__/random_package/src/index.tsx

```diff
@@ -32,5 +32,5 @@ export const getContrastRatingsOfTokensWithTextContrastRequirements = (): void =
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 export const RandomComponent = () => {
-  return <TextareaAutosize />;
+  return <Tab />;
```

Comment:
todo: I suggest using [jest module mocks](https://jestjs.io/docs/manual-mocks) to handle this instead. Happy to chat in person if you have questions.

Reply:
I'm going to leave that as out of scope, I just swapped the test for another 3rd party component because we don't have that dependency anymore.

Reply:
Leaving as out of scope because I just swapped the component for another 3rd party component.